deploy/crds/compliance.openshift.io_v1alpha1_compliancesuite_cr.yaml

```diff
@@ -7,12 +7,14 @@ spec:
   scans:
       - name: workers-scan
         profile: xccdf_org.ssgproject.content_profile_moderate
+        scanType: Node
         content: ssg-ocp4-ds.xml
         contentImage: quay.io/jhrozek/ocp4-openscap-content:latest
         nodeSelector:
             node-role.kubernetes.io/worker: ""
       - name: masters-scan
         profile: xccdf_org.ssgproject.content_profile_moderate
+        scanType: Node
```

Comment:
@JAORMX @jhrozek I thought about making Node the default if you don't specify.

Reply:
That sounds good to me.